### DISCUSSION TITLE
请问一下怎么加载Enhancer 模型或者打开Enhancer 呢 请求大佬解答谢谢

### DISCUSSION THREAD

**Discussion by funcxs** [lenML/Speech-AI-Forge#46] [问答 | Q&A]
请问一下怎么加载Enhancer 模型或者打开Enhancer 呢 请求大佬解答谢谢

**Comment by zhzLuke96** [comment]
打开？webui里你勾选上增强就会自动加载

**Reply by funcxs** [reply]
请问如果使用api怎么传入增强参数呢 我是小白麻烦大佬解答

**Reply by zhzLuke96** [reply]
> 请问如果使用api怎么传入增强参数呢 我是小白麻烦大佬解答

现目前（0.6.1-rc）api中只有 google api 实现了人声增强

```json
{
  // ...
  "enhancerConfig": {
    "enabled": false,
    "model": "resemble-enhance",
    "nfe": 32,
    "solver": "midpoint",
    "lambd": 0.5,
    "tau": 0.5
  }
}
```

其中 `enhancerConfig.enabled` 为 true 即可调用 enhancer

其余参数可以打开 `/docs` 查看

> 其他接口对接 enhancer 还在开发中，0.6.1 应该会支持所有api调用enhancer
